guides/source/active_record_multiple_databases.md

```diff
@@ -583,3 +583,23 @@ Rails also doesn't support automatic load balancing of replicas. This is very
 dependent on your infrastructure. We may implement basic, primitive load balancing
 in the future, but for an application at scale this should be something your application
 handles outside of Rails.
+
+### Working With Multiple Databases On Rails Console
+
+By default, Rails console connects solely to your default database. Consider the following configuration.
```

Comment:
This is only true when eager loading is off. In production all connections are established on boot, however as noted below `connected_to?` can only return true for the current active connection. Is it not for "was this connection established at some point?". For that you should use `connection_pool_list` to check if there is a connection in the pool matching the one you're looking for (however this isn't a very useful feature as it's really an implementation detail and not something apps should be checking for).